keps/prod-readiness/sig-apps/2599.yaml

```diff
@@ -1,3 +1,30 @@
 kep-number: 2599
 alpha:
   approver: "@ehashman"
+owning-sig: sig-apps
```

Comment:
I think these changes are not necessary, not for PRR metadata file.

Reply:
TY. Updated :)